README

```diff
@@ -1,11 +1,15 @@
 A collection of scripts to be used for aiding the development of Mate.
 
+- build/build.mk buils a given mate project and all his dependencies (assuming
```

Comment:
_"[…] and all **its** dependencies […]"_

Reply:
@cwendling Thanks for point me the typos.